src/tests/ll_verification.py

```diff
@@ -2401,6 +2401,206 @@ def wait_for_AUX_ADV_IND_end():
+        elif i < 3:
+            # Unexpected/wrong event received
+            success = False
+
+    # Keep connection open until we have (at least) 100 LL data PDUs from Lower Tester
+    while True:
+        LLDataCount = 0
+        for packet in packets.fetch('LL_DATA_PDU'):
+            if packet.idx == lowerTester:
+                LLDataCount += 1
+        if LLDataCount >= 100:
+            break
+
+        # Wait for the expected time to get the remaining packets based on the connection interval
+        transport.wait(math.ceil((100 - LLDataCount)*(initiator.intervalMax*1.25)))
+
+    # Check LL_DATA_PDUs
+    # The IUT sends and receives data using the LE 1M PHY
+    for packet in packets.fetch(packet_filter=('LL_DATA_PDU')):
+        success = success and packet.phy == '1M'
+
+    # The Lower Tester receives no further advertising packets while maintaining the connection
+    firstLLDataPDUTs = packets.find('LL_DATA_PDU').ts
+    lastAdvIndTs = packets.findLast('ADV_IND').ts
+    success = success and lastAdvIndTs <= firstLLDataPDUTs
+
+    # Disconnect with "Remote User Terminated Connection"
+    status = disconnect(transport, upperTester, connectionHandle, 0x13, 200)
+    success = success and (status == 0)
+
+    # Wait for disconnect to fully complete
+    transport.wait(500)
+
+    # Flush system before the next test steps
+    packets.flush()
+    flush_events(transport, lowerTester, 100)
+    flush_events(transport, upperTester, 100)
+
+    # Set advertising parameters and start advertising again
+    success = preamble_ext_advertising_parameters_set(transport, upperTester, Handle, Properties, PrimMinInterval, PrimMaxInterval,
+                                                      PrimChannelMap, OwnAddrType, PeerAddrType, PeerAddress, FilterPolicy, TxPower,
+                                                      PrimAdvPhy, SecAdvMaxSkip, SecAdvPhy, Sid, ScanReqNotifyEnable, trace)
+
+    if not success:
+        return False
+
+    success = success and preamble_ext_advertise_enable(transport, upperTester, Advertise.ENABLE, [Handle], [0], [0], trace)
+    if not success:
+        return False
+
+    # Wait until the simulation time is just after an ADV_IND packet has transmitted (ie. within T_IFS so a response can be transmitted)
+    # Returns the ADV_IND packet or None if it fails to find one within an advertising interval
+    def wait_for_ADV_IND_end():
+        timeOut = math.ceil(advInterval*0.625+10)*1000
+        checkInterval = 100 # Note: Has to be less than T_IFS
+        advIndPacket = None
+        while timeOut > 0:
+            lastPacket = packets.findLast(packet_filter=('ADV_IND'))
+            if lastPacket:
+                # Check that simulation time is just after the ADV_IND has ended (so we can transmit a response)
+                simulation_time = transport.get_last_t()
+                airTime = (1 + 4 + 2 + len(lastPacket) + 3)*8
```

Comment:
I think `get_packet_air_time` could be used here

Reply:
Yup - that function wasn't in yet when I made this. I'll rebase and use it instead